gcc/rust/lint/rust-lint-marklive.h

```diff
@@ -107,6 +107,11 @@ class MarkLive : public MarkLiveBase
       }
   }
 
+  void visit (HIR::UnsafeBlockExpr &expr) override
+  {
+    expr.get_block_expr ()->accept_vis (*this);
+  }
+
```

Comment:
It will be more convenient there is an auto walk visit class for HIR in order to avoid writing template walking code. I'll work on this.